freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -173,6 +173,12 @@ public Group groupAdd(String group) throws FreeIpaClientException {
         return (Group) invoke("group_add", flags, params, Group.class).getResult();
     }
 
+    public void deleteGroup(String group) throws FreeIpaClientException {
+        List<String> flags = List.of(group);
+        Map<String, Object> params = Map.of();
+        invoke("delete_group", flags, params, Object.class);
```

Comment:
does this work? all the other ipa commands are in the form `noun_verb` like `user_del`. I would have expected this to be `group_del` instead of `delete_group`